packages/storybook/stories/va-number-input.stories.jsx

```diff
@@ -4,7 +4,7 @@ import { getWebComponentDocs, propStructure, StoryDocs } from './wc-helpers';
 const numberInputDocs = getWebComponentDocs('va-number-input');
 
 export default {
-  title: 'V1 Components/Number input',
+  title: 'Deprecated/Number input V1',
```

Comment:
I'm wondering if we need to add `V1` to this because there isn't any other "number" component, right?

![Screenshot 2024-05-13 at 2 04 33 PM](https://github.com/department-of-veterans-affairs/component-library/assets/872479/6056ff0d-ee00-4f5a-a2db-931196ca41a5)

Reply:
@jamigibbs - good catch, fixed!